airflow/providers/amazon/aws/hooks/lambda_function.py

```diff
@@ -40,30 +40,31 @@ class LambdaHook(AwsBaseHook):
 
     def __init__(
         self,
-        function_name: str,
-        log_type: str = 'None',
-        qualifier: str = '$LATEST',
-        invocation_type: str = 'RequestResponse',
         *args,
         **kwargs,
     ) -> None:
-        self.function_name = function_name
-        self.log_type = log_type
-        self.invocation_type = invocation_type
-        self.qualifier = qualifier
         kwargs["client_type"] = "lambda"
         super().__init__(*args, **kwargs)
 
-    def invoke_lambda(self, payload: str) -> str:
+    def invoke_lambda(self, function_name: str, **kwargs):
```

Comment:
It would be better if the parameters were spelled out, especially because they now also have typing information. But previously, those parameters also had defaults.

Reply:
@malthe, I agree. I've made the recommended changes and added defaults.